src/components/transactions/EditTransactionModal/index.tsx

```diff
@@ -35,27 +35,57 @@ interface Props {
 
 const initializeAccountingLinesIncoming = (transaction: TransactionProps): AccountingLine[] => {
   if (transaction.direction === Direction.Swap) {
-    const accountingLinesSwapIn = transaction?.accountingLines?.filter(accountingLine => accountingLine.accountingType && AccountingTransactionType.Swap === accountingLine.accountingType) || [];
```

Comment:
I wanted a default AccountingLine to be present when the modal loads. In the process of adding it I streamlined the functions.